doc-content/enterprise-only/installation/supported-repos-con.adoc

```diff
@@ -0,0 +1,22 @@
+[id='supported-repos-con_{context}']
+
+= {PRODUCT} supported repositories
+You can use Git repositories and Apache Maven with {PRODUCT}.
+
+== Git repositories
+Git repositories are used internally within {CENTRAL} to store all processes, rules, and other artifacts that are created in the authoring environment. Git is a distributed version control system. It implements revisions as commit objects. When you commit your changes into a repository, a new commit object in the Git repository is created. When you create a project in {CENTRAL} it is added to the Git repository connected to {CENTRAL}.
```

Comment:
"it is added to the Git repository connected to {CENTRAL}."

it is added to the Git repository that is connected to {CENTRAL}.

Reply:
Done